aten/src/ATen/native/quantized/cudnn/Conv.cpp

```diff
@@ -449,14 +455,8 @@ at::Tensor PackedConvWeightCudnn<kSpatialDim>::apply_impl(
       at::MemoryFormat::ChannelsLast);
   // requantization
   // out_int8 = act_int8 * weight_int8 * act_scale * w_scale / output_scale
-  // TODO:  note we will remove the int_repr() in a subsequent PR, so we can move the computations for
-  // the multipliers into the helper function
-  auto act_scale = act.q_scale();
-  auto weight_scale = orig_weight_.q_scale();
-  auto requantize_multiplier = act_scale * weight_scale / output_scale;
-  auto bias_multiplier = 1.0 / (act_scale * weight_scale);
   apply_impl_helper<kReluFused>(
```

Comment:
qq: why do we need apply_impl_helper? can we move the code in apply_impl_helper to apply_impl?